modin/engines/base/frame/partition_manager.py

```diff
@@ -302,6 +302,8 @@ def concat(cls, axis, left_parts, right_parts):
             A new BaseFrameManager object, the type of object that called this.
         """
         if type(right_parts) is list:
+            # filtering empty frames
+            right_parts = [o for o in right_parts if o.size != 0]
```

Comment:
@dchigarev please give a short explanation.

Reply:
`df.loc["row"]["col"]` assignment in case of one row will trigger concatenation of empty frame.

`np.array` with partitions of empty ModinFrame has a shape `(0,)`, but `np.concatenate` can concatenate arrays only if its shapes at specified axis are equals, so filtering empty frames to avoid concat error

added that comment at the last commit